docs/manual/developer/08_content_tests.md

```diff
@@ -15,9 +15,44 @@ To add a script to be checked with mypy use the `mypy_test` macro in `tests/CMak
 
 ### Unit Tests
 
-`ctest` is used to run the unit tests for the `ssg` python package that is in the repo.
+`ctest` is used to run the unit tests for the `ssg` python package that is in the repo. But you need to build content and `ctest` isn't able to run unit tests with differed versions of python. To run only unit tests with differed version of python for the `ssg` python package is recommended to use `tox`.
```

Comment:
I don't understand what you mean by "But you need to build content and ..." . If you mean that you need to build SCAP content in order to run unit tests then that's not true.

Reply:
If I understand [documentation](https://complianceascode.readthedocs.io/en/latest/manual/developer/02_building_complianceascode.html#testing) correctly? It means that the unit tests are part of the `ctest` test suite, and before running `ctest` you need to execute the `./build_product` command to build the content.  For `tox`, just clone the repo, install `tox` and run it.

Reply:
But we're talking about Python unit tests of the `ssg` Python package. For them to run, you don't have to build anything, it's enough to just configure:

```
cd build 
rm -rf *
cmake ..
ctest -R python-unit-ssg-module
```

Reply:
Okay, I'll add it to the documentation. This information was missing from the documentation.